apps/idam/migration-base/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I dont think this is needed? above` base/kustomization.yaml` should be fine

Reply:
As discussed, to use migrate-base as an interim to review how Fluxv2 interacts. Not long term, long term will be ../../base